cg/meta/report/balsamic.py

```diff
@@ -62,22 +62,24 @@ def get_sample_metadata(
                 million_read_pairs=million_read_pairs, sample_metrics=sample_metrics
             )
         return self.get_panel_metadata(
-            sample=sample,
             million_read_pairs=million_read_pairs,
             sample_metrics=sample_metrics,
             analysis_metadata=analysis_metadata,
         )
 
-    @staticmethod
     def get_panel_metadata(
-        sample: Sample,
+        self,
         million_read_pairs: float,
         sample_metrics: BalsamicTargetedQCMetrics,
         analysis_metadata: BalsamicAnalysis,
     ) -> BalsamicTargetedSampleMetadataModel:
         """Returns a report metadata for BALSAMIC TGS analysis."""
+        bed_version: BedVersion = self.status_db.get_bed_version_by_filename(
+            analysis_metadata.config.panel.capture_kit
+        )
+        bed: Bed = self.status_db.get_bed_by_id(bed_version.bed_id) if bed_version else None
```

Comment:
I get a type warning on this line in my editor, might be a setup issue on my part though
```
Argument of type "Column" cannot be assigned to parameter "bed_id" of type "int" in function "get_bed_by_id"
  "Column" is incompatible with "int"
```